### DISCUSSION TITLE
Lob lob lob - und ein Feature request :-)

### DISCUSSION THREAD

**Discussion by thorsten-gehrig** [frickelzeugs/FingerprintDoorbell#21] [Ideas]
Hi
erstmal viel Lob - ich finde das project echt cool und habe schon eine "Klingel" gebaut und seit heute Mittag fest installiert :-)
(Ich nutze das nicht als Klingelknopf - sondern dediziert als Leser)

Und hier mein Feature-Request: hast du schonmal darüber nachgedacht KNX-IP mit einzubauen?
Ich denke MQTT durch KNX zu ersetzen (bzw. alternativ oder beides) sollte machbar sein - fertige Libraries für KNX-over-IP gibt es schon: https://github.com/envy/esp-knx-ip

Gruß
Thorsten

**Comment by rasor89** [comment]
Ich schließe mich dem Feature-Request an: Eine Anbindung an KNX wäre super! 

Mfg
Christian

**Comment by henfri** [comment]
Ich auch :-)

**Comment by nobbi1991** [comment]
Bin auch interessiert :)

**Comment by gfa4711** [comment]
Ich wäre auch daran interessiert! :-)
